index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<html>
+
+    <head>
+        <title>Chat Room</title>
+    </head>
+
+    <body>
+        
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+        <header>
+            <h1>Chat Room</h1>
+        </header>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
```

Comment:
Home is not required to be a link here, as you are on the Home page already (there is no need to have the option to navigate to the current page). Just have it as free text